.circleci/config.yml

```diff
@@ -344,6 +344,7 @@ workflows:
             branches:
               only:
                 - master
+                - feature/976-logging-of-graphql-queries-in-google
```

Comment:
@pylipp please remove `feature/976-logging-of-graphql-queries-in-google`  before merge :)